addSkill.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Aggiungi Competenza</title>
+        <meta charset="UTF-8">
+        <meta name="author" content="Leonardo Morganti">
+        <meta name="description" content="Gestione Competenze">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <table>
+            <tr>
+                <td><a href=modifica.html><button type=button>Go back</button></a></td></td>
+            </tr>
+            <tr>
+                <td><select width name="employee" id="employee">
+                    <option value="" selected="selected" disabled="disabled" >Seleziona Dipendente</option>
+                </select></td>
+            </tr>
+            <tr>
+                <td><input type="text" name="nomeSkill" id="nomeSkill" placeholder="nuova competenza"></td>
```

Comment:
scrivo qui ma vale per tutta la pull request

ok che siamo italiani, parliamo italiano e ci aspettiamo il sito web con le label italiane, però la lingua franca per programmare è l'inglese quindi tutto ciò che è programmazione o modellazione (html) deve riportare nomi in inglese